core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -220,8 +220,8 @@ class SearchService(
     backlogActor ! FileUpdate(f, symbols)
   }(workerEC)
 
-  def shutdown(): Unit = {
-    db.shutdown()
+  def shutdown()(implicit ec: ExecutionContext): Unit = {
+    db.shutdown
```

Comment:
there are a couple of ECs in that code and we wanted to make sure we were using the right one at the right time. It needs a rewrite.